cardigan/.storybook/preview.js

```diff
@@ -54,4 +55,7 @@ export const parameters = {
       </DocsContainer>
     ),
   },
+  nextRouter: {
+    Provider: RouterContext.Provider,
```

Comment:
https://storybook.js.org/addons/storybook-addon-next-router <-- yay! allows us to use storybook for components relying on Router.